oci/oci.go

```diff
@@ -40,6 +40,8 @@ const (
 	SystemdCgroupsManager = "systemd"
 	// ContainerExitsDir is the location of container exit dirs
 	ContainerExitsDir = "/var/run/crio/exits"
+	// Base directory for container attach sockets
```

Comment:
Standard is to start with the constant 
AttachSocketDir is the base directory for container attach sockets

Reply:
Fixed